bin/craft_comment_with_dependency_diff

```diff
@@ -32,7 +32,8 @@ print_tree() {
 %s
 \`\`\`
 
-</details>" "$1"
+</details>
+" "$1"
```

Comment:
Question (❓): Why not applying the same change on the `print_header()` function, how would doing so change the comment? 🤔